pi-manage

```diff
@@ -586,6 +586,25 @@ def create(name, scope, action):
     return r
 
 
+@policy_manager.command
+def create_from_file(name, filename):
+    """
+    create a new policy from a JSON-file
+    """
+    import json
+
+    f = open(filename, 'r')
+    contents = f.read()
+    f.close()
+    params = json.loads(contents)
```

Comment:
So the problem was that `ast.literal_eval()` requires all values to be quoted. Therefore:
```
  "active": true,
  "check_all_resolvers": false,
```

Must be:
```
  "active": "true",
  "check_all_resolvers": "false",
```